src/simulator/base.py

```diff
@@ -0,0 +1,27 @@
+from object import Object, ObjectId
+from position import Position
+from shape import Shape
+from object_registry import ObjectRegistry
+from robot import Robot
+import numpy as np
+
+RadioID = str  # type aliasing, since we might change RadioID composition
+
+
+class Base(Object):
+    def __init__(self, oid: ObjectId, name: str, shape: Shape, position: Position,
+                 owner_object: Object, registry: ObjectRegistry, rid: RadioID, error: int):
+        super().__init__(oid, name, shape, position, owner_object, registry)
+        self.radio_id = rid
+        self.error = error
```

Comment:
I don't know if there are other uses of this variable besides being the scale for normal distribution or not, but since it must have a positive value, I suggest using a name to indicate this fact (like standard_deviation, width_of_distribution, etc) or mentioning it in the documentation.